controller/config.go

```diff
@@ -73,6 +72,13 @@ const (
 	// detault
 	MongoMemoryProfile = "mongo-memory-profile"
 
+	// LogPruneIntervalDays is the pruning interval for log entries.
+	LogPruneIntervalDays = "logs-prune-interval-days"
```

Comment:
can this please take a duration suffix, rather than encoding a specific unit in the name? you can use time.ParseDuration to convert into a time.Duration